gslib/utils/text_util.py

```diff
@@ -203,7 +210,27 @@ def InsistAsciiHeaderValue(header, value):
       value,
       'Invalid non-ASCII value (%s) was provided for header %s.\nOnly ASCII '
       'characters are allowed in headers other than x-goog-meta- and '
-      'x-amz-meta- headers' % (value, header))
+      'x-amz-meta- headers' % (repr(value), header))
+
+
+def InsistOnOrOff(value, message):
```

Comment:
1) So, it looks like this got added 4 extra times, for a total of 5 definitions. Not sure how that happened.

2) Please add a docstring, to whichever of them you decide to keep, saying that `value` should be of type unicode.

Reply:
Removed duplicates and added docstrings to a couple functions, including `InsistOnOrOff()`